src/audio/google_audio_post_processing_mock.c

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2022 Google LLC.
+//
+// Author: Kehuang Li <kehuangli@google.com>
+
+#include "google_audio_post_processing.h"
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "sof/lib/alloc.h"
+#include "ipc/topology.h"
+
+#define GOOGLE_AUDIO_POST_PROCESSING_SAMPLE_RATE_HZ 48000
+
+struct GoogleAudioPostProcessingState {
+	uint8_t config[8];
+	int num_channels;
+	int num_frames;
+	int volume[SOF_IPC_MAX_CHANNELS];
+};
+
+GoogleAudioPostProcessingState *GoogleAudioPostProcessingCreate(void)
+{
+	struct GoogleAudioPostProcessingState *s = rballoc(
+		0, SOF_MEM_CAPS_RAM, sizeof(GoogleAudioPostProcessingState));
+
```

Comment:
preffered way would be call `rballoc` later after `s` declaration. ANd also check for return value.

Reply:
What is wrong with instantiating and declaring in the same line? But yes we do need a nullity check here

Reply:
Added a check.

Reply:
@cujomalainey nothing wrong but the code is harder to read. Anyhow, not a blocker anway.